src/app/application.cpp

```diff
@@ -660,4 +661,7 @@ void Application::validateCommandLineParameters()
 {
     if (m_commandLineArgs.portableMode && !m_commandLineArgs.profileDir.isEmpty())
         throw CommandLineParameterError(tr("Portable mode and explicit profile directory options are mutually exclusive"));
+
+    if (m_commandLineArgs.portableMode && m_commandLineArgs.relativeFastresumePaths)
+        throw CommandLineParameterError(tr("Portable mode and explicit relative fastresume mode options are mutually exclusive"));
```

Comment:
> IMO, we should silently ignore...

Wouldn't that create an impression that `--relative-fastresume` is used somehow? Let's print a warning instead?